src/json-formatter.html

```diff
@@ -4,7 +4,10 @@
     <span class="key" ng-if="hasKey"><span class="key-text">{{key}}</span><span class="colon">:</span></span>
     <span class="value">
       <span ng-if="isObject()">
-        <span class="constructor-name">{{getConstructorName(json)}}</span>
+        <span class="json-title" ng-if="!!title" ng-bind="title"></span>
```

Comment:
instead of making a new element (`json-title`) it's better to update `getConstructorName` method to return `scope.title` if it exists otherwise does what it's doing currently.